apps/vessel-history/features/vessels/sources/tmt.slice.ts

```diff
@@ -23,15 +24,31 @@ const extractValue: (valueItem: ValueItem[]) => string | undefined = (valueItem:
 const sortAuthorization = (a: Authorization, b: Authorization) =>
   a.originalStartDate > b.originalStartDate ? 1 : -1
 
+const sortIuu = (a: Iuu, b: Iuu) =>
+  a.originalStartDate > b.originalStartDate ? 1 : -1
+
 const getHistoryField = (historyField: AnyValueList[]): VesselFieldHistory<any> => ({
   byCount: [],
   byDate: historyField.reverse().map((field) => ({ ...field, source: VesselAPISource.TMT })),
 })
+
+const getIuuHistory = (historyField: Iuu[]): VesselFieldHistory<any> => ({
+  byCount: [],
+  byDate: historyField.reverse().map((field: Iuu) => ({
```

Comment:
The list is being reversed two times here and also in [line 63](https://github.com/GlobalFishingWatch/frontend/pull/1701/files?diff=split&w=1#diff-a5d4d737dcf53b760de185d8c27625c6b6301c237dae439071fc7ae6213074acR63). I'd remove the reverse on both places.